Sources/Tools/MaxComponent/plGUIComponents.cpp

```diff
@@ -4442,8 +4441,8 @@ bool plGUIProgressCtrlComponent::Convert(plMaxNode *node, plErrorMsg *pErrMsg)
         // so to avoid breaking old formats, if we can't grab an animObjInterface, we just grab the key
         // of the master mod of our node, like we would've before
         plAGMasterMod   *master = node->GetAGMasterMod();
-        hsTArray<plKey> keys;
-        keys.Append( master->GetKey() );
+        std::vector<plKey> keys;
+        keys.emplace_back(master->GetKey());
         ctrl->SetAnimationKeys( keys, ENTIRE_ANIMATION_NAME );
```

Comment:
Could we use an initializer list here, eg

```suggestion
        ctrl->SetAnimationKeys( { master->GetKey() }, ENTIRE_ANIMATION_NAME );
```